newspaper/parsers.py

```diff
@@ -247,7 +246,7 @@ def getAttribute(cls, node, attr=None):
         if attr:
             attr = node.attrib.get(attr, None)
         if attr:
-            attr = HTMLParser().unescape(attr)
+            attr = unescape(attr)
```

Comment:
I would have preferred the style of 
```
import html
html.unescape(..)
```

But it's not like this repository does a good job of setting standards :( I'm going to accept this and merge it for sake of removing the deprecating warnings hitting everyone and probably refactor later